src/client-services/github.client-service.test.tsx

```diff
@@ -0,0 +1,69 @@
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+import {
+  axiosConfig,
```

Comment:
I know you didn't make these but can you make the following small changes.

Name change:
axiosConfig -> axiosInstance or getAxiosInstance
getReposList -> getRepositoryList
getRepoInfo -> getRepositoryInformation

IMO, the names are more clear and avoids abbreviations.